src/components/ContractTab/ContractLogin.vue

```diff
@@ -0,0 +1,233 @@
+        maximumFractionDigits: 2,
+    });
+});
+
+watchEffect(() => {
+    if (isLoggedIn.value) {
+        fetchUserBalance();
+    }
+});
+
+function handleWalletButtonClick() {
+    if (!isLoggedIn.value) {
+        showLoginModal.value = true;
+    }
+}
+
+function logout() {
+    useAccountStore().logout();
+}
+
+function goToAccountPage() {
+    $router.push({
+        name: 'address',
+        params: {
+            address: address.value,
+        },
+    });
+}
+
+function copyAddress() {
+    copyToClipboard(address.value);
+
+    $q.notify({
+        position: 'bottom',
+        message: $i18n.t('components.header.address_copied'),
+        color: 'green',
+    });
+}
+
+async function fetchUserBalance() {
+    const response = await indexerApi.get(
+        `/account/${address.value}/balances`,
+    );
+    const tlos = response.data?.results?.find(({ contract }: { contract: string }) => contract === '___NATIVE_CURRENCY___');
+    userSystemTokenBalanceWei.value = tlos?.balance ?? '0';
+}
+</script>
+
+<template>
+<OutlineButton
+    text-color="default"
+    class="c-app-header-wallet__main-button"
+    v-bind="$attrs"
+    @click="handleWalletButtonClick"
+>
+    <template v-if="!isLoggedIn">
+        {{ $t('components.header.connect_wallet') }}
+
+        <q-icon
+            name="far fa-circle"
+            size="10px"
+            class="c-app-header-wallet__circle-icon"
+        />
+    </template>
+
+    <template v-else>
+        {{ prettyIdentity }}
+        <q-icon
+            name="fas fa-circle"
+            size="10px"
+            class="c-app-header-wallet__circle-icon c-app-header-wallet__circle-icon--logged-in"
+        />
+        <q-menu>
+            <q-list>
+                <q-item>
+                    <div class="c-app-header-wallet__account-actions-container">
+                        <div>
+                            <OutlineButton
+                                text-color="primary"
+                                :icon-only="true"
+                                :aria-label="$i18n.t('components.header.copy_address')"
+                                @click="copyAddress"
+                            >
+                                <q-icon name="far fa-copy" size="14px" />
+
+                                <q-tooltip>
+                                    {{ $i18n.t('components.header.copy_address') }}
+                                </q-tooltip>
+                            </OutlineButton>
+
+                            <OutlineButton
+                                text-color="primary"
+                                :icon-only="true"
+                                class="q-ml-sm"
+                                @click="goToAccountPage"
+                            >
+                                <q-icon name="far fa-address-card" size="14px" />
+
+                                <q-tooltip>
+                                    {{ $i18n.t('components.header.goto_address_details') }}
```

Comment:
just for consistency
```suggestion
                                    {{ $t('components.header.goto_address_details') }}
```